### PR TITLE
Packify SNOW

### PR DESCRIPTION
**Pull request by liorblob**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

For https://github.com/demisto/etc/issues/22557

### CODE REVIEW

File: Packs/ServiceNow/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1 @@
+## [Unreleased]
```
Comment (Itay4) [inline Packs/ServiceNow/CHANGELOG.md]: delete this

**Comment by liorblob** [comment]
@Itay4 Yes, as part of the next issue which will also require documentation

**Comment by Itay4** [comment]
what about test playbooks?

**Comment by liorblob** [comment]
they're in NonCircleTests

**Pull request merged**